cime_config/temp_tuvx_config.json

```diff
@@ -121,27 +49,32 @@
             "cross section": "O3",
             "vertical profile": "O3",
             "vertical profile units": "molecule cm-3"
-         },
-         {
-            "name": "aerosols",
-            "type": "aerosol",
-            "optical depths": [2.40e-01, 1.06e-01, 4.56e-02, 1.91e-02, 1.01e-02, 7.63e-03,
-                               5.38e-03, 5.00e-03, 5.15e-03, 4.94e-03, 4.82e-03, 4.51e-03,
-                               4.74e-03, 4.37e-03, 4.28e-03, 4.03e-03, 3.83e-03, 3.78e-03,
-                               3.88e-03, 3.08e-03, 2.26e-03, 1.64e-03, 1.23e-03, 9.45e-04,
-                               7.49e-04, 6.30e-04, 5.50e-04, 4.21e-04, 3.22e-04, 2.48e-04,
-                               1.90e-04, 1.45e-04, 1.11e-04, 8.51e-05, 6.52e-05, 5.00e-05,
-                               3.83e-05, 2.93e-05, 2.25e-05, 1.72e-05, 1.32e-05, 1.01e-05,
-                               7.72e-06, 5.91e-06, 4.53e-06, 3.46e-06, 2.66e-06, 2.04e-06,
-                               1.56e-06, 1.19e-06, 9.14e-07],
-            "single scattering albedo": 0.99,
-            "asymmetry factor": 0.61,
-            "550 nm optical depth": 0.235
          }
       ]
    },
    "photolysis reactions": {
-      "O2+hv->O+O": {
+      "jo3_a": {
+         "__reaction": "O3+hv->O2+O(1D)",
```

Comment:
Have you tested this config file? I thought I had setup the photolysis rates to expect a field called "name", which would be this `__reaction`, I think

Reply:
Yeah, I'm updating for this and some other changes in the main TUV-x branch after my dev branch got merged in

Reply:
done!